superset-frontend/plugins/plugin-chart-pivot-table/src/react-pivottable/TableRenderers.jsx

```diff
@@ -24,7 +24,11 @@ import { PivotData, flatKey } from './utilities';
 import { Styles } from './Styles';
 
 const parseLabel = value => {
-  if (typeof value === 'number' || typeof value === 'string') {
+  if (typeof value === 'string') {
+    if (value === 'metric') return t('metric');
```

Comment:
We do not need to change the parseLabel function. Instead, we can simply translate the METRIC_KEY constant in superset-frontend/plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx:58

Reply:
When I put t('metric') for the METRIC_KEY constant, I don't see the trnaslation in the chart

Reply:
Because you need to rebuild pot file, then rebuild po files and build messages.json again.

Reply:
Ok, let me test it.

Reply:
It works on me, but anyway you need to build .pot file and .po files.
Here's instructions: https://superset.apache.org/docs/contributing/translations/

Reply:
Oh okay, I did it but not so easy to work with this generation because that remove some translation

Reply:
I don't know why I have still warning for prettier as I launch this command before push `npx prettier --update ../superset/translations/**/LC_MESSAGES/*.json`

Reply:
Hello.
I think we can ignore that warning.

Reply:
Okay because, I got a job in error becuse of that

Reply:
Where did you receive information that the messages.json files need to be refactored with Prettier? As I understand it, they can be pushed as builded. And your previous commits with changed messages.json passed all tests.

Reply:
I builded your mo files on my and found that after building everything is fine. With or without Prettier refactoring, there are no warnings. 
However, the messages.json files that you pushed are still getting warnings from Prettier on my. Can you please check your translation using the `npm run check-translation` command on your computer before pushing and make it works?
Maybe you don't need to refactor messages.json by prettier?

Reply:
It's what I did and I have this return 
![Capture d’écran du 2023-02-01 17-53-41](https://user-images.githubusercontent.com/104373599/216184245-1bdc155b-d5ca-4185-a6b4-e25c9fb900cb.png)